<!-- This file is based on nissan_interface.launch -->
<!-- Author: Matt Bunting -->
<!-- Note: I made this for testing to simply suppress debug messages from other services -->

<launch>

<!-- This interface is generic to each vehicle, and sends the raw data to ROS for decoding in can_to_ros -->
<node pkg="can_to_ros" type="vehicle_interface" name="vehicle_interface" output="screen" />

<!-- Converts target gap and speed into button press commands to be read by vehicle_interface (Nissan specific) -->
<node pkg="can_to_ros" type="nissan_target_speed_to_acc_buttons" name="nissan_target_speed_to_acc_buttons" output="screen" />

<!-- Publishes ROS topics that are to be used by subscribers of either recording or control -->
<node pkg="can_to_ros" type="subs" name="subs" output="screen" />

</launch>
